src/components/playButton/PlayButton.styles.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components'
+import theme from '../../theme/theme'
+
+export const ButtonWrapper = styled.div`
+  width: 160px;
```

Comment:
Can you use the sizing from the space.ts file. You can import `xxl` and multiply by 2 to get 160.

You also need to include the mobile sizing, use the media query from `mediaQueries.ts`
for example: 

```
const ButtonWrapper = styled.div`
   // mobile first, so mobile width and height goes here.
   ${mediaQueries.md} {
    // CSS goes here for screens medium and above 
  }
`